frontend/src/components/organisms/form-sections/OrganisationSection/TechnicalTeamCard.js

```diff
@@ -1,22 +1,63 @@
-import React, { useContext, useMemo } from 'react';
+import React, { useContext, useEffect, useMemo } from 'react';
 import { Card } from '../../../molecules/Card';
-import SelectInput from '../../../atoms/inputs/SelectInput';
 import { FormContext } from '../../../templates/Form';
+import { TextInputWithSuggestions } from '../../../molecules/TextInputWithSuggestions';
+import RadioInput from '../../../atoms/inputs/RadioInput';
+import Button from '../../../atoms/hyperTexts/Button';
 import TextInput from '../../../atoms/inputs/TextInput';
-import TextInputWithSuggestions from '../../../molecules/TextInputWithSuggestions';
+import Alert from '../../../atoms/Alert';
+import Link from '../../../atoms/hyperTexts/Link';
+import { HideSectionsContext } from '../../../templates/Form/HideSectionsContainer';
+
+const typeOptions = [
+  { id: 'software_company', label: 'Votre éditeur de logiciel' },
+  { id: 'internal_team', label: 'Votre équipe de développeurs' },
+  { id: 'other', label: 'Autre' },
+];
 
 export const TechnicalTeamCard = ({ editorList = [] }) => {
   const {
     disabled,
     onChange,
+    isUserEnrollmentLoading,
     enrollment: { technical_team_type, technical_team_value },
   } = useContext(FormContext);
 
+  const { setReadyForNextSteps } = useContext(HideSectionsContext);
+
+  const areInputValid = useMemo(() => {
```

Comment:
Premature optimisations with useMemo make the code harder to read and maintain, is his not more performant for light computation due to useMemo overhead.
 See the deep dive section at https://beta.reactjs.org/learn/you-might-not-need-an-effect#caching-expensive-calculations
 
Just use boolean checks : 
```ts
const areInputValid =
    technical_team_type === 'internal_team' || !!technical_team_value;```